playbooks/roles/nginx/templates/assets.j2

```diff
@@ -26,6 +26,14 @@ server {
         proxy_pass $agreements_s3_url;
     }
 
+    # Hack to force g7 communications to the old bucket to preserve upload times
+    location ^~ /g-cloud-7-updates/communications {
```

Comment:
It's a different kind of rule. The ones that start with a `~` on it's own are regular expression rule. This one (`^~`) is a prefix rule that overrides regular expression rules.

http://nginx.org/en/docs/http/ngx_http_core_module.html#location